src/components/r4-track-actions.js

```diff
@@ -9,17 +9,6 @@ template.innerHTML = `
 `
 
 export default class R4TrackActions extends HTMLElement {
-	get id() {
-		return this.getAttribute('id')
-	}
-	set id(str) {
-		if (str) {
-			this.setAttribute('id', str)
-		} else {
-			this.removeAttribute('id')
-		}
-	}
```

Comment:
not needed because `id` is also available as prop on HTMLElement. Could argue that we shouldn't use `id` for this rather `track-id`.